src/ralph/data_importer/mixins.py

```diff
@@ -27,28 +29,27 @@ def __new__(cls, name, bases, attrs):
         update_fields = []
         for name, field in new_class.fields.items():
             update_fields.append((name, field))
+            field_name = '{}_str'.format(name)
+            field_params = dict(
+                column_name='{}_str'.format(field.column_name),
+                attribute=field.attribute,
+                readonly=True
+            )
             if isinstance(field.widget, widgets.ForeignKeyWidget):
-                update_fields.append((
-                    '{}_str'.format(name),
-                    fields.Field(
-                        column_name='{}_str'.format(field.column_name),
-                        attribute=field.attribute,
-                        widget=ExportForeignKeyStrWidget(),
-                        readonly=True
-                    )
-                ))
+                field_params['widget'] = ExportForeignKeyStrWidget()
+            elif isinstance(field.widget, ManyToManyThroughWidget):
+                field_params['widget'] = ExportManyToManyStrTroughWidget(
+                    model=field.widget.model,
+                    related_model=field.widget.related_model,
+                    through_field=field.widget.through_field,
+                )
             elif isinstance(field.widget, widgets.ManyToManyWidget):
-                update_fields.append((
-                    '{}_str'.format(name),
-                    fields.Field(
-                        column_name='{}_str'.format(field.column_name),
-                        attribute=field.attribute,
-                        widget=ExportManyToManyStrWidget(
-                            model=field.widget.model
-                        ),
-                        readonly=True
-                    )
-                ))
+                field_params['widget'] = ExportManyToManyStrWidget(
+                    model=field.widget.model
+                )
+            else:
+                continue
```

Comment:
if this field does not fulfil any of the conditions (`isinstance` of `ForeignKeyWidget`, `ManyToManyThroughWidget` or `ManyToManyWidget`) we don't want to process this field in any special way (here: creating second column with "str" version of this field).